pulp_rpm/app/tasks/publishing.py

```diff
@@ -88,6 +88,8 @@ def prepare_metadata_files(self, content, folder=None):
         )
 
         for repo_metadata_file in repo_metadata_files:
+            if repo_metadata_file.unsupported_metadata_type:
```

Comment:
Leave a comment about why this is needed (because of historical repository versions)

Reply:
Done. (Though I feel like my sentence structure is needlessly complex somehow).

Reply:
It's fine!